brew install micro --HEAD stopped working

A month ago the --HEAD installation option stopped working.

`brew install micro --HEAD`

```
==> Upgrading micro
  HEAD-3737979 -> HEAD-968f5ba 
==> make build-tags
Last 15 lines from /Users/User/Library/Logs/Homebrew/micro/01.make:
2024-08-29 12:40:23 +0000

make
build-tags

git fetch --tags
GOOS=darwin GOARCH=arm64 go generate ./runtime
From https://github.com/zyedidia/micro
 ! [rejected]          nightly     -> nightly  (would clobber existing tag)
 * [new tag]           v2.0.14     -> v2.0.14
 * [new tag]           v2.0.14-rc1 -> v2.0.14-rc1
make: *** [fetch-tags] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ install: generate
 install-all: install
 
 fetch-tags:
-	git fetch --tags
+	git fetch --tags --force
 
 generate:
 	GOOS=$(shell go env GOHOSTOS) GOARCH=$(shell go env GOHOSTARCH) go generate ./runtime

